integration/fargate_test.go

```diff
@@ -0,0 +1,173 @@
+// +build integration
+
+package integration_test
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/dlespiau/kube-test-harness"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	. "github.com/weaveworks/eksctl/integration/runner"
+	"github.com/weaveworks/eksctl/pkg/utils/names"
+)
+
+var _ = Describe("(Integration) Fargate", func() {
+
+	// Fargate is not supported in us-west-2 yet
+	const region = "ap-northeast-1"
+
+	deleteCluster := func(clusterName string) {
+		cmd := eksctlDeleteCmd.WithArgs(
+			"cluster", clusterName,
+			"--verbose", "4",
+			"--region", region,
+		)
+		Expect(cmd).To(RunSuccessfully())
+	}
+
+	type fargateTest struct {
+		clusterName string
+		kubeTest    *harness.Test
+	}
+
+	setup := func(ft *fargateTest, createArgs ...string) {
```

Comment:
Is there a reason for keeping these functions as variables instead of extracting them as functions at the end of this file?

Reply:
Mainly to avoid polluting the package scope with these functions. All integration tests are part of the same `integration_test` package.

Reply:
Right, lets leave it like that then and when we move it to its own package we extract them